code/game/objects/effects/decals/Cleanable/fuel.dm

```diff
@@ -6,14 +6,14 @@
 	anchored = 1
 	var/amount = 1 //Basically moles.
 
-/obj/effect/decal/cleanable/liquid_fuel/Initialize(turf/newLoc, amt = 1, nologs = 0)
-	if(!nologs)
-		log_game("[amt] units of liquid fuel have spilled in [AREACOORD(newLoc.loc)].")
-		message_admins("[amt] units of liquid fuel have spilled in [ADMIN_VERBOSEJMP(newLoc.loc)].")
+/obj/effect/decal/cleanable/liquid_fuel/Initialize(mapload, amt = 1, logs = TRUE)
+	if(logs)
+		log_game("[amt] units of liquid fuel have spilled in [AREACOORD(loc.loc)].")
+		message_admins("[amt] units of liquid fuel have spilled in [ADMIN_VERBOSEJMP(loc.loc)].")
 	amount = amt
 
 	//Be absorbed by any other liquid fuel in the tile.
-	for(var/obj/effect/decal/cleanable/liquid_fuel/other in newLoc)
+	for(var/obj/effect/decal/cleanable/liquid_fuel/other in loc)
 		if(other != src)
 			src.amount += other.amount
 			qdel(other)
```

Comment:
. =.. () should probably be top of init and spread() should probably be in either late init or a timer to avoid chain spawning hell during init